packages/auth/amplify_auth_cognito_dart/lib/src/state/event/fetch_auth_session_event.dart

```diff
@@ -114,9 +114,8 @@ class FetchAuthSessionRefresh extends FetchAuthSessionEvent {
 
   @override
   String? checkPrecondition(FetchAuthSessionState currentState) {
-    if (currentState.type == FetchAuthSessionStateType.refreshing ||
-        currentState.type == FetchAuthSessionStateType.fetching) {
-      return 'Credentials are already being fetched...';
+    if (currentState.type == FetchAuthSessionStateType.refreshing) {
```

Comment:
Have you tested this?

Reply:
Ok - I recall needing this with some of the session logic, but I know other things have changed in that code path since then.

Reply:
I had added the second part of the check by mistake. Refreshes are always initiated while the current state is `fetching`. This still debounces concurrent refresh attempts.